packages/hash/external-services/kratos/identity.schema.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
This is the ory kratos identity schema, which defines what is stored on individual identities. Currently we are only storing email addresses on kratos identities, down the line we may expand this to include user shortnames to enable logging in with a shortname for example. More information about kratos identity schemas can be found [here](https://www.ory.sh/docs/guides/manage-identities/identity-schema).

My instinct is the majority of user information will be stored on the user entity in the HASH graph down the line (rather than on the identities stored in kratos).